libs/deal-ticket/src/components/info-market.tsx

```diff
@@ -0,0 +1,170 @@
+import pick from 'lodash/pick';
+import omit from 'lodash/omit';
+import type { DealTicketQuery_market } from './__generated__/DealTicketQuery';
+import BigNumber from 'bignumber.js';
+
+export interface InfoProps {
+  market: DealTicketQuery_market;
+}
+
+export const Info = ({ market }: InfoProps) => {
+  const headerClassName =
+    'text-h4 font-bold uppercase text-black dark:text-white';
+  return (
+    <div className="p-16 flex flex-col gap-32">
+      <div className="flex flex-col gap-16">
+        <p className={headerClassName}>{t('Market data')}</p>
+        <AccordionPanel
+          key="fees"
+          title={t('Current Fees')}
+          content={
+            <>
+              <MarketInfoTable data={market.fees.factors} asPercentage={true} />
+              <p className="text-ui-small">
+                {t(
+                  'All fees are paid by price takers and are a % of the trade notional value. Fees are not paid during auction uncrossing.'
+                )}
+              </p>
+            </>
+          }
+        />
+        <AccordionPanel
+          key="market-data"
+          title={t('Market data')}
+          content={
+            <MarketInfoTable
+              data={market.data}
+              decimalPlaces={market.decimalPlaces}
+            />
+          }
+        />
+      </div>
+
+      <div className="flex flex-col gap-16">
+        <p className={headerClassName}>{t('Market specification')}</p>
+        <AccordionPanel
+          title={t('Key details')}
+          key="details"
+          content={
+            <MarketInfoTable
+              data={pick(
+                market,
+                'name',
+                'decimalPlaces',
+                'positionDecimalPlaces',
+                'tradingMode',
+                'state'
+              )}
+            />
+          }
+        />
+        <AccordionPanel
+          title={t('Instrument')}
+          key="instrument"
+          content={
+            <MarketInfoTable
+              data={{
+                product: market.tradableInstrument.instrument.product,
+                ...market.tradableInstrument.instrument.product.settlementAsset,
+              }}
+            />
+          }
+        />
+        <AccordionPanel
+          title={t('Risk Model')}
+          key="risk-model"
+          content={
+            <MarketInfoTable data={market.riskFactors} unformatted={true} />
+          }
+        />
+        {(market.priceMonitoringSettings?.parameters?.triggers ?? []).map(
+          (trigger, i) => (
+            <AccordionPanel
+              key={`trigger-${i}`}
+              title={t(`Price Monitoring Trigger #${i + 1}`)}
+              content={<MarketInfoTable data={trigger} />}
+            />
+          )
+        )}
+      </div>
+    </div>
+  );
+};
+
+export interface RowProps {
+  key: string;
+  value: any;
+  decimalPlaces?: number;
+  asPercentage?: boolean;
+  unformatted?: boolean;
+}
+
+export const renderRow = ({
```

Comment:
question (non-blocking): any reason this is a function instead of a react component? looks a bit non-reacty imo.

Reply:
I had an issue with the keys, but updated in last commit.